src/CordovaError/CordovaError.js

```diff
@@ -50,10 +50,8 @@ CordovaError.EXTERNAL_TOOL_ERROR = 1;
  */
 CordovaError.prototype.getErrorCodeName = function () {
     for (var key in CordovaError) {
```

Comment:
```suggestion
     for (const key of Object.keys(CordovaError)) {
```

I noticed this before but forgot to add it to my review. I'm so sorry about that.

With the change proposed above, we should be preserving the exact behavior of the changed code. That is, iterating only over own keys.